### ISSUE TITLE
Consider moving (Third Edition) to a subtitle

### ISSUE THREAD

**Issue by palemieux** [w3c/ttml1#352]
Per https://github.com/tobie/specref/issues/463, I suggest considering moving  (Third Edition) to a subtitle

**Comment by skynavga** [comment]
I do not agree, as this would make it inconsistent with the titling of prior editions.

**Comment by palemieux** [comment]
@skynavga How do you address the issue raised at https://github.com/tobie/specref/issues/463? I am happy with alternatives. The status quo is undesirable however.

**Comment by skynavga** [comment]
The fix is obvious: change IMSC to refer to specific versions, and not the generic spec URL. We went through TTML2 normative references not long ago to make sure we always referred to a specific document with a permanent URL. See https://github.com/w3c/ttml2/issues/457.

As for changing the title of TTML1, I cannot accept a change there.

**Comment by nigelmegitt** [comment]
I agree, if we want to reference a specific version we should use that version's URL. On the flip side if we want a generic URL then there needs to be a usable name for the referenced spec independent of edition and version. That latter thing seems to be what this issue is asking to facilitate.

**Comment by palemieux** [comment]
> n the flip side if we want a generic URL then there needs to be a usable name for the referenced spec independent of edition and version. 

Yes, precisely. Editions are intended to be very minor revisions and so presumably there should be a root name that does not include the edition number.

**Comment by nigelmegitt** [comment]
I've reassigned the pull request to @plehegar pending further information about the need and urgency of making this change.
